proofpoint-threat-response/info.json

```diff
@@ -0,0 +1,883 @@
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "text",
+        "name": "username",
+        "parameter_type": "header_param",
+        "description": "Specify the Username used to access the Proofpoint Threat Response server to connect and perform automated operations.",
+        "tooltip": "Specify the Username used to access the Proofpoint Threat Response server to connect and perform automated operations."
+      },
+      {
+        "title": "Password",
+        "required": true,
+        "editable": true,
+        "visible": true,
+        "type": "text",
+        "name": "password",
+        "parameter_type": "header_param",
+        "description": "Specify the Password used to access the Proofpoint Threat Response server to connect and perform automated operations.",
+        "tooltip": "Specify the Password used to access the Proofpoint Threat Response server to connect and perform automated operations."
+      },
+      {
+        "title": "Verify SSL",
+        "name": "verify_ssl",
+        "required": false,
+        "editable": true,
+        "visible": true,
+        "type": "checkbox",
+        "value": true,
+        "tooltip": "Specifies whether the SSL certificate for the server is to be verified. By default, this option is set to True.",
+        "description": "Specifies whether the SSL certificate for the server is to be verified. By default, this option is set to True."
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "get_list",
+      "title": "Get List",
+      "category": "investigation",
+      "annotation": "get_list",
+      "description": "Gets items for the specified list.",
+      "output_schema": [],
+      "parameters": [
+        {
+          "name": "list_id",
+          "title": "List ID",
+          "type": "text",
+          "description": "Specify the ID of list to fetch the details.",
+          "tooltip": "Specify the ID of list to fetch the details.",
+          "required": true,
+          "editable": true,
+          "visible": true
+        }
+      ],
+      "enabled": true
+    },
+    {
+      "operation": "add_to_list",
+      "title": "Add To List",
+      "category": "investigation",
+      "annotation": "add_to_list",
+      "description": "Adds a member to the specified list.",
+      "output_schema": [],
+      "parameters": [
+        {
+          "name": "list_id",
+          "title": "List ID",
+          "type": "text",
+          "description": "Specify the ID of list to add provided members.",
+          "tooltip": "Specify the ID of list to add provided members.",
+          "required": true,
+          "editable": true,
+          "visible": true
+        },
+        {
+          "name": "indicator",
+          "title": "Indicator",
+          "type": "text",
+          "description": "Specify the Indicator values. Value can be IP Address, URLs. Domains or file hashes. e.g. 192.168.1.1,192.168.1.2",
```

Comment:
Replace single dot with comma after the `URLs` and `file hashes` with `File Hashes`.
Make changes in description as well in tooltip.